api/restHandler/PipelineConfigRestHandler.go

```diff
@@ -3277,3 +3309,155 @@ func (handler PipelineConfigRestHandlerImpl) PipelineNameSuggestion(w http.Respo
 	}
 	writeJsonResp(w, err, suggestedName, http.StatusOK)
 }
+
+func validatejson(jsondoc pipeline.TemplateRequest, chartVersion string) bool {
```

Comment:
use better name and extract this method and all depending methods in a separate file